frontend/amundsen_application/static/images/icons/Application.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
What does this icon look like?

I would expect it to be named 'application.svg' instead.

Reply:
This icon looks like a gear with a square around it to represent generic applications (designed by Knowl). I renamed the file to be lowercase now, I noticed there was a mixture of some upper and some lower for the icon naming and chose wrong 😄